### DISCUSSION TITLE
Looking to Get Started with CNCF Contributions – Seeking Guidance

### DISCUSSION THREAD

**Discussion by Nikhil713** [cncf/mentoring#1313] [Q&A] [answered]
Hello,
I hope you're doing well! My name is Nikhil Babu, and I’m currently pursuing my Master’s in Computer Science degree in Germany. I have some experience working with cloud technologies, and I’m eager to deepen my understanding and contribute meaningfully to the cloud-native ecosystem.

CNCF’s mission to advance cloud-native computing strongly aligns with my interests, and I’d love to get involved with open-source projects in this space. I would really appreciate some guidance on how I can get started:

- Are there beginner-friendly projects or tasks I can explore?
- Are there areas that are currently looking for contributors?
- Any recommended resource to better understand CNCF workflows and projects?

I’m excited to learn, grow, and contribute alongside this incredible community. Looking forward to your guidance!

**Comment by krook** [comment, marked as answer]
Hi @Nikhil713,

Welcome! The best place to get started is join our [Slack workspace](https://communityinviter.com/apps/cloud-native/cncf) and check out the pinned links in the `#cncf-new-contributors` channel there.

**Reply by Nikhil713** [reply]
Hi @krook, joined the Slack workspace. Thanks for the quick response!

**Reply by nate-double-u** [reply]
Also, check out https://contribute.cncf.io/contributors/getting-started/

**Comment by FARDEEN-785** [comment]
Hello,
I’m Mohd Fardeen Khan, a BTech student in Computer Science. I have a good understanding of Python and Flask, and I’m really interested in learning and contributing to the cloud-native ecosystem.

CNCF’s work in advancing cloud-native computing really inspires me, and I want to get involved with open-source projects in this space. I’d love some guidance on:

Beginner-friendly projects or tasks to start with.
Areas where contributors are currently needed.
Resources to better understand CNCF workflows and projects
.
I’m eager to learn, grow, and contribute to this amazing community. Looking forward to your suggestions!

with regards
Mohd Fardeen khan

**Reply by krook** [reply]
Hi @FARDEEN-785 please check out the links above to get started

**Reply by FARDEEN-785** [reply]
> Hi @FARDEEN-785 please check out the links above to get started

Hi @krook ,  just joined the slack workspace 
Thanks for the quick response!
